src/k8s-engine/lima.ts

```diff
@@ -745,8 +747,11 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
    */
   protected async limaWithCapture(this: Readonly<this>, ...args: string[]): Promise<string> {
     args = this.debug ? ['--debug'].concat(args) : args;
-    const { stdout } = await childProcess.spawnFile(LimaBackend.limactl, args,
-      { env: LimaBackend.limaEnv, stdio: ['ignore', 'pipe', console] });
+    const { stdout, stderr } = await childProcess.spawnFile(LimaBackend.limactl, args,
+      { env: LimaBackend.limaEnv, stdio: ['ignore', 'pipe', 'pipe'] });
+    const formatbreak = stderr || stdout ? '\n' : '';
+
+    console.log(`> limactl ${ args.join(' ') }${ formatbreak }${ stderr }${ stdout }`);
```

Comment:
Do we want a `try`/`catch`/`throw` here to log the command on error (and re-throw the error)?

Reply:
Initially I was trying to avoid changing any function behaviors, but this could be a good place for a try/catch.  Should I do that?

Reply:
Sounds like it might be a good idea.